release_notes/47-release-notes.adoc

```diff
@@ -478,4 +480,27 @@ This release provides fixes for the following security issues:
 * zlib: Out-of-bounds pointer arithmetic in `inftrees.c` link:https://access.redhat.com/security/cve/CVE-2016-9840[(CVE-2016-9840)]
 * krb5: Kerberos RC4-HMAC-MD5 checksum vulnerability link:https://access.redhat.com/security/cve/CVE-2025-3576[(CVE-2025-3576)]
 
+[id="about-release-475_{context}"]
+== About release version 4.7.5
+
+*Release date*: {ga-date-475}
+
+This release of {product-title-short} 4.7 provides the following bug fixes:
+
+//ROX-29320
+* Before this update, Scanner V4 falsely claimed not to support RHEL 10, although this was always the case. Scanner V4 now correctly indicates that it supports RHEL 10.
```

Comment:
> Before this update, Scanner V4 falsely claimed not to support RHEL 10, although this was always the case.

I feel like this may be misinterpreted to mean it was always the case it never supported RHEL 10. Can we change it to something like:

Before this update, Scanner V4 falsely claimed to not support RHEL 10, although it always supported RHEL 10.